src/main/scala/com/advancedtelematic/campaigner/actor/CampaignScheduler.scala

```diff
@@ -68,9 +68,11 @@ class CampaignScheduler(director: DirectorClient,
       schedule(devices).pipeTo(self)
 
     case BatchToSchedule(devices) if devices.isEmpty =>
-      parent ! CampaignComplete(campaign.id)
-      // TODO: Should move to finished
-      context.stop(self)
+      campaigns.updateStatus(campaign.id).onComplete { _ =>
+        log.debug(s"Completed campaign: ${campaign.id}")
```

Comment:
Isn't this logged in `CampaignSupervisor` already?

Reply:
True. Guess it's just a leftover statement I used to debug tests. I believe it won't hurt much if leave it there :)